R/generateFilterValues.R

```diff
@@ -4,12 +4,18 @@
 #' Calculates numerical filter values for features.
 #' For a list of features, use [listFilterMethods].
 #'
+#' @importFrom purrr map_dfr imap map
+#' @importFrom magrittr %>%
+#' @importFrom dplyr group_by arrange mutate summarise rename bind_rows mutate_if
+#' @importFrom tibble as_tibble tibble
 #' @template arg_task
 #' @param method ([character])\cr
 #'   Filter method(s), see above.
 #'   Default is \dQuote{randomForestSRC.rfsrc}.
 #' @param nselect (`integer(1)`)\cr
 #'   Number of scores to request. Scores are getting calculated for all features per default.
+#' @param ensemble.method ([character])\cr
+#'   Ensemble filter method to use. Can only be used with >= 2 filter methods.
```

Comment:
Should be consistent with wrapper -- character or logical?

Reply:
Nothing is finished yet :)